### PR TITLE
Adding AddRealmRoleToGroup and DeleteRealmRoleFromGroup methods

### PR DESCRIPTION
**Pull request by dlisin**
These methods will allow for managing realm roles for groups. Currently it is only supported for users

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -175,6 +175,8 @@ type GoCloak interface {
 	DeleteRealmRole(token string, realm string, roleName string) error
 	AddRealmRoleToUser(token string, realm string, userID string, roles []Role) error
 	DeleteRealmRoleFromUser(token string, realm string, userID string, roles []Role) error
+    AddRealmRoleToGroup(token string, realm string, groupID string, roles []Role) error
```
Comment (SVilgelm) [inline README.md]: Please use a tab symbol here

Comment (dlisin) [reply README.md]: Thanks, for pointing this.
Fixed

**Comment by SVilgelm** [comment]
There is an issue in Keycloak. Looks like the changed something and now we cannot import real.json, it fails:
```
20:28:44,367 INFO  [org.keycloak.services] (ServerService Thread Pool -- 70) KC-SERVICES0050: Initializing master realm
20:28:45,972 WARN  [org.keycloak.services] (ServerService Thread Pool -- 70) KC-SERVICES0005: Unable to import realm gocloak from file /tmp/gocloak-realm.json.: java.lang.NullPointerException
	at org.keycloak.keycloak-services@8.0.2//org.keycloak.url.DefaultHostnameProvider.resolveUri(DefaultHostnameProvider.java:83)
	at org.keycloak.keycloak-services@8.0.2//org.keycloak.url.DefaultHostnameProvider.getScheme(DefaultHostnameProvider.java:38)
	at org.keycloak.keycloak-server-spi@8.0.2//org.keycloak.models.KeycloakUriInfo.<init>(KeycloakUriInfo.java:46)
```

The importing works from web ui, but it fails on the start-up. I imported and then exported the realm from the web ui, but the start-up script also cannot import the exported real,

**Comment by dlisin** [comment]
@SVilgelm yes, it is looks like some problem was introduced in Keycloak v8.0.2
When I try to run tests locally on v8.0.1 everything working fine

**Comment by SVilgelm** [comment]
https://github.com/Nerzal/gocloak/pull/141

**Comment by SVilgelm** [comment]
Released in v4.6.0: https://github.com/Nerzal/gocloak/releases/tag/v4.6.0

**Pull request merged**